### PR TITLE
fixed !!2!! bugs with campfire

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: code/ATMOSPHERICS/hvac/spaceheater.dm [hunk 1]
```diff
@@ -157,12 +157,14 @@
 		var/obj/item/stack/sheet/wood/woody = I
 		woodnumber = min(woodnumber,woody.amount)
 		woody.amount -= woodnumber
+		if(woody.amount <= 0)
```
Comment (unid15) [inline code/ATMOSPHERICS/hvac/spaceheater.dm]: Why not woody.use(woodnumber)

Comment (ghost) [inline code/ATMOSPHERICS/hvac/spaceheater.dm]: a fair point

**Comment by ghost** [comment]
fixes #7045

**Comment by ghost** [comment]
now fixes #7043.

**Pull request merged**